data/ch10/ch10_load.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Please set the following variables
+DSBULK_PATH=/PATH/TO/dsbulk-1.3.4
+DEST_KS=movies_dev
+
+############################################
+# Automated data loading
+# Assuming you set DSBULK_PATH and DEST_KS
+############################################
+
+# collect directory variables
+OLDWD="`pwd`"
+SCRIPTIR="`dirname $0`"
+cd "$SCRIPTDIR"
+DATADIR="`pwd`"
+
+# determine which dsbulk to use, in the case you already have it installed
+DSBULK="`which dsbulk`"
+if [ -z "$DSBULK" ]; then
+    if [ ! -f "$DSBULK_PATH/bin/dsbulk" ]; then
+        echo "Please set DSBULK_PATH variable to top-level path of DSBulk distribution"
+        echo "  or add directory with dsbulk to PATH, like, 'PATH=...dsbulk-1.3.4/bin:\$PATH'"
+        exit 1
+    fi
+fi
+DSBULK=$DSBULK_PATH/bin/dsbulk
+
+############################################
+# load the vertices
+############################################
+echo "Loading vertices into the graph $DEST_KS"
+
+echo "Extracting data files for loading..."
+tar xvzf movies_dev_data.tar.gz
+
+echo "Loading vertices into the graph $DEST_KS"
+$DSBULK load -k $DEST_KS -t Movie -url "$DATADIR/Movie.csv" -header true
```

Comment:
we may remove the `$DATADIR/` as we changed to `$DATADIR` already. This is the same for all other scripts.